queue_consumer.go

```diff
@@ -198,30 +199,37 @@ type result struct {
 	success bool
 }
 
+// AWS SQS defaults
 const (
 	defaultDelayAfterReceiveError          = 5 * time.Second
 	defaultReceiveVisibilityTimeoutSeconds = 30
 	defaultExtendVisibilityBySeconds       = 60
 	defaultExtendVisibilityEvery           = 30 * time.Second
 	defaultDeleteAccumulatorTimeout        = 250 * time.Millisecond
 	defaultDeleteMessageDrainTimeout       = time.Second
+	defaultReceiveMessageWaitSeconds       = 20
+	defaultMessagesBatchSizeLimit          = 10
+)
 
-	// AWS maximums
-	awsBatchSizeLimit                = 10
-	defaultReceiveMessageWaitSeconds = 20
+const (
+	// default number of concurrent goroutines processing SQS messages by calling Consumer.handler
+	defaultJobWorkersCount = 10
```

Comment:
Nitpick:  Why not group this constant with the other defaults above?

Reply:
I can do that, this const is not aws config, it is only used by this sqsconsumer library, thought I might keep it separate.